docs/management/watcher-ui/index.asciidoc

```diff
@@ -1,8 +1,56 @@
 [role="xpack"]
 [[watcher-ui]]
-== Watcher UI
+== Watcher
+
+Watcher is an {es} feature that you can use to create actions based on 
```

Comment:
I think in general we're not using "Watcher" as much anymore. If you look on https://www.elastic.co/subscriptions and https://www.elastic.co/products/stack/alerting it's not mentioned there. 

I recommend using the alert-features attribute from https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc

For example:
```suggestion
The {alert-features} enable you to create actions based on 
```